src/istgtcontrol.c

```diff
@@ -84,6 +84,7 @@
 
 #define	MAX_LINEBUF 4096
 #define	UCTL_CHAP_CHALLENGE_LEN 1024
+#define REPLICA_ID_LEN 32
```

Comment:
`REPLICA_ID_LEN` should be consistent with libcStor. Instead of defining it here, let's use it's definition from `zrepl_prot.h` only.

Reply:
Updated